### PR TITLE
Set session created time properly

### PR DESCRIPTION
**Pull request by Henry0422**
https://nanopay.atlassian.net/browse/CPF-1678

### CODE REVIEW

**Review by jlhughes — APPROVED** [review]

**Review by adamvy — APPROVED** [review]

File: src/foam/box/SessionServerBox.java [hunk 1]
```diff
@@ -53,6 +54,7 @@ public void send(Message msg) {
         session = new Session();
         session.setId(sessionID == null ? "anonymous" : sessionID);
         session.setRemoteHost(req.getRemoteHost());
+        session.setCreated(new Date());
```
Comment (kgrgreer) [inline src/foam/box/SessionServerBox.java:57]: Why is this necessary? Here is the definition of the created property:
    {
      class: 'DateTime',
      name: 'created',
      factory: function() { return new Date(); },
      javaFactory: 'return new Date();'
    },

which should do this already. Do we have a bug in FOAM?

Comment (adamvy) [reply src/foam/box/SessionServerBox.java:57]: Factories are not triggered at object creation time, but at first access, so this will return the time that the created property is first accessed.

**Comment by kgrgreer** [comment]
Oh, yah. So we could also just do:
session.getCreated();

**Comment by adamvy** [comment]
Indeed.  I'm good with either personally.  .getCreated() is shorter, .setCreated(new Date()) is clearer.
